grafana-plugin/src/state/context/directPaging.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, FC, createContext, PropsWithChildren, useCallback } from 'react';
```

Comment:
We don't use Context anywhere in the app, that's why we have `MobX` ... why do you need it? This could be very well the state of the component, you don't even a separate store file for this I believe.